openedx/core/djangoapps/programs/tests/test_utils.py

```diff
@@ -10,20 +11,22 @@
 from edx_oauth2_provider.tests.factories import ClientFactory
 from provider.constants import CONFIDENTIAL
 
+# This gets around an error about conflicting models documented at
+# https://code.djangoproject.com/ticket/22280. Importing the models from
+# `lms.djangoapps.certificates.models` raises a RuntimeError.
+from certificates.models import GeneratedCertificate  # pylint: disable=import-error
```

Comment:
optional: not to be a stickler here (ok, maybe to be a stickler), we still have this reference to physical models from the certificates app.  It seems like it would be possible to expose the constants you're referencing via a simple variable or object in the API module, which would have the added benefit of avoiding the conflicting model workaround, but if it's not as simple as that, not a dealbreaker.